lib/ui/views/messenger/layouts/components/discussion_search_bar.dart

```diff
@@ -184,75 +195,13 @@ class _DiscussionSearchBarState extends ConsumerState<DiscussionSearchBar> {
           textAlign: TextAlign.center,
           controller: searchController,
           autocorrect: false,
-          maxLines: 2,
```

Comment:
Why have you remove 2 lines ? if user wants to see all the address in the same time, perhaps it's better